app/components/channel_drawer_list/channel_drawer_list.js

```diff
@@ -13,13 +13,16 @@ import {
     View
 } from 'react-native';
 import {injectIntl, intlShape} from 'react-intl';
+import AwesomeIcon from 'react-native-vector-icons/FontAwesome';
+import MaterialIcon from 'react-native-vector-icons/MaterialIcons';
```

Comment:
Why are we mixing Font Awesome and Material icons? I think we should stick with Font Awesome to keep everything consistent